invenio_oauthclient/templates/semantic-ui/invenio_oauthclient/_macros.html

```diff
@@ -0,0 +1,43 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2015-2018 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% macro render_field(field, icon="", placeholder='', autofocus=False) %}
+  <div class="field {% if field.errors %} error{% endif %}">
+    {%- set extras = dict(autofocus="") if autofocus else dict() %}
+    {{field(class_="form-control", placeholder=placeholder or _(field.label.text | string), **extras)}}
+    {%- if icon %}
+    <i class="{{icon}}" aria-hidden="true" ></i>
+    {%- endif%}
+  </div>
+{% endmacro %}
+
+
+{% macro form_errors(form) %}
```

Comment:
Same as for @kprzerwa were you able how the errors are rendered?

Reply:
I can't render form_errors because it is used only in the `signup.html` and to view that file I need to setup the Github authorization.

Reply:
https://github.com/inveniosoftware/invenio-accounts/issues/325

Reply:
Created dedicate issue: https://github.com/inveniosoftware/invenio-oauthclient/issues/209